### DISCUSSION TITLE
Status of the project?

### DISCUSSION THREAD

**Discussion by alastairs** [dotnet/spark#1125] [Q&A]
Hi there, I wonder if it's possible to provide an official "update" on the status of this project. It seems to be updated very infrequently, it hasn't had a release since June 2022, it does not yet support .NET 6.0 (over twelve months after its release and with barely weeks to go until .NET Core 3.1 falls out of support), new issues are not responded to before weeks or months have passed, pull requests have been hanging around since June 2019, and the SPIP seems to be dead in the water. The last ["status update?" post](https://github.com/dotnet/spark/discussions/973) was left in September 2021 and went unanswered. 

It's clear Microsoft is not putting a lot of resource into this project, and given it's a .NET Foundation project perhaps that's to be expected. It's presence in the `dotnet` platform organisation, though, gives a stamp of officialdom it wouldn't otherwise have. Please could we have some statement from the maintainers on what to expect from this project in the future? Is it "done", is there a roadmap for future updates, ...?

**Comment by simonhoss** [comment]
If this is not supported anymore does anyone know a good alternative for having data analysis with dotnet?

**Comment by GeorgeS2019** [comment]
It seems the .NET6 upgrade will happen, [the support is back](https://github.com/dotnet/spark/issues/1139)
